src/scss/_mixins.scss

```diff
@@ -107,10 +107,10 @@
 	}
 
 	.o-cookie-message__button {
-		@include oButtons(
-			$size: 'big',
-			$theme: 'primary'
-		);
+		@include oButtonsContent((
+			'size': 'big',
+			'theme': 'primary'
```

Comment:
'primary' and 'secondary' are now button "types". Themes are "inverse", "mono", "b2c", which can be used to modify the look of the two button types